gen/write.go

```diff
@@ -21,6 +21,10 @@ import (
 	"github.com/ogen-go/ogen/ogenregex"
 )
 
+func init() {
+	imports.LocalPrefix = "github.com/ogen-go/ogen"
+}
```

Comment:
I don't like this but it is to maintain the current grouping of imports in generated files. If you're not that bothered about that I'd be in favour of removing it as I'm not a fan of side-effects such as setting global vars is 3rd party packages.

Reply:
Let's remove it. Import grouping is not that important, I think.

Reply:
Have removed it and regenerated everything.